### ISSUE TITLE
android < 10 support

### ISSUE THREAD

**Issue by Bluscream** [rom1v/sndcpy#35]
I know that earlier android versions dont support capturing system audio so i would like to know if any of the following allows capturing it anyway:

- sndcpy installed as system app
- giving special permissions to sndcpy through adb
- root
- xposed/magik

**Comment by Bluscream** [comment]
If any of the mentioned work it would be super nice to have it implemented so theres atleast some hope for pre android 10 devices

**Comment by Genxster1998** [comment]
@rom1v https://gitlab.com/vijai/screenrecorder , this has internal audio recording function for pie and below

**Comment by rom1v** [comment]
@Genxster1998 In the app, when you select "default source" (for internal audio), you get this warning:
![audio](https://user-images.githubusercontent.com/543275/91558408-907f7780-e936-11ea-9c2b-a4af36bb8824.png)

I tried to record with my OnePlus 7 Pro, it just uses the microphone (when I talk, it's recorded in the video).

**Comment by Genxster1998** [comment]
Strange , For me , it works with Mx Player Video. , It does have mic and internal audio mix channel but it does record video Sound playing in Mx Player , other option is also there ,magisk one

**Comment by rom1v** [comment]
> it does record video Sound playing in Mx Player

If you talk while you record, is your voice recorded?
If you set the volume to 0 on the device, is it recorded?

**Comment by Genxster1998** [comment]
@rom1v yeah as it says ,it does mix internal sound capture with mic and  external sound capture that also depend on device volume level ,what about other one ,magisk ,is it same ?
